### ISSUE TITLE
Vulkano nightly build failed due to Shaderc compilation

### ISSUE THREAD

**Issue by Eliah-Lakhin** [vulkano-rs/vulkano#1423]
Due to a Travis report Vulkano `master` build is failing for Nightly Rust on Linux: https://travis-ci.org/github/vulkano-rs/vulkano/jobs/733905213 .

The error log showing glsl compilation error on this line of the Shader-Include example: https://github.com/vulkano-rs/vulkano/blob/master/examples/src/bin/shader-include/main.rs#L62

Shaderc error messages are:

```
   = help: message: 3 compilation errors:
           shader.glsl:8: error: '#line' : must by followed by an integral literal
           shader.glsl: error: shader.glsl:-1: '#line' : unexpected tokens following directive
           0:0: error: '' :  syntax error, unexpected RETURN
```

I investigated the code, and didn't find any issues in implementation. This is an example that was implemented long time ago and it worked well so far. Also, I don't see any changes in Vulkano-shaders subproject that may cause this error.

After some debugging of Vulkano-shaders I managed to figure out that the error message is coming from the `shaderc-rs` entry function: https://github.com/vulkano-rs/vulkano/blob/master/vulkano-shaders/src/codegen.rs#L184

And, what's more important, it happening in the Rust Nightly only. I managed to reproduce this bug, and can confirm that it relevant to Rust Nightly and irrelevant to Rust Stable. The parameter values passing to `shaderc`'s "compile_into_spirv" function exactly the same in Stable and Nightly. So, my suspicion is the bug somewhere in the Shaderc-rs crate(or maybe even in the shaderc itself).

I also can confirm that the issue with includes appearing in my personal project too which is using includes extensively. And in Nightly build only too.

This is the minimal example of glsl code causing this error:

```glsl
#version 450
#include <common.glsl>
```

No matter what's inside "common.glsl". If "common.glsl" was properly resolved Shaderc throughs the error mentioned above(about the `#line` directive that is not even used in the snippet).

---

My suggestion to admins(@AustinJ235) is to turn off Travis CI nightly build for a while since it blocking incoming PRs reviews and merging. I can't do it myself, I don't have access to Travis.

And I will write a message to Shaderc-rs developers to help me investigating the bug.

**Comment by Eliah-Lakhin** [comment]
Related Shaderc-rs conversation: https://github.com/google/shaderc-rs/issues/86

**Comment by AustinJ235** [comment]
Any pull request can modify the travis ci through the config in the repo.

**Comment by Eliah-Lakhin** [comment]
@AustinJ235 Yes, but I can't merge even my own PRs if they don's pass tests: https://github.com/vulkano-rs/vulkano/pull/1422 And there is git hook prevent pushing to `master` directly.

**Comment by Eliah-Lakhin** [comment]
ok, I think I figured out how to modify Travis from PR's feature branch. It passed Travis tests https://travis-ci.org/github/vulkano-rs/vulkano/builds/734112201 but seems like it tested irrelevant commit to the PR https://github.com/vulkano-rs/vulkano/pull/1424 Something is broken on the Travis side(or maybe I'm doing something wrong). To ensure this is not an accident I tried to push dummy commits to the branch and even recreated the PR from the fresh branch, but no luck.
